### PR TITLE
Guess Who Project W5

### PR DESCRIPTION
**Pull request by CoffeeDoodles**
https://elainesguesswhogame.netlify.app/

github: https://github.com/CoffeeDoodles/project-guess-who

I struggled through this project, js and I have not yet become friends.

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -253,91 +256,130 @@ const setSecret = () => {
...
+  }
+
+ }
 
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
-  // Show the correct alert message for different categories
-  if (group === 'accessories') {
+  let category = currentQuestion.attribute
+   if (category === 'accessories') {
```
Comment (Frivridna) [inline code/script.js:308]: Great decision to reuse 'category' here, and remove the prewritten group. It makes it easier to follow the code!

**Review by Frivridna — CHANGES_REQUESTED** [review]
Hello Elaine! It was our very first solo programming week with JavaScript on board I think you did a fantastic job! I know that you struggled through this one <3 and I want to give applause for fighting and focusing on getting it to function properly. 
 
It's great that the player is able to restart the game after having guessed on a person, and you basically solved almost all of the problems that you describe in your README.
 
In order to fulfill the blue level, you should also give the player feedback with alerts. I wrote quite an extensive comment directly under the filterCharacters() where I am trying to help out as much as I can.... though it might help us both if a teacher looks at the code as well. 

Let me know if you have any questions regarding my comments.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
As Caroline pointed in her review you need to inform your user if a guess was good or not using alert.

File: code/script.js (hunk 1 above)
Comment (CoffeeDoodles) [reply code/script.js]: By changing currentQuestion.attribute to currentQuestion.category, all my alerts started to function. :)

**Review by KarolCislowski — APPROVED** [review]
Great job 😸

**Pull request closed without merging**